File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        l =''
        for i in s:
            if i.isalpha() or i.isnumeric():
                if i.isalpha():
                    l+=i.lower()
                else:
                    l+=i
        i = 0
        j = len(l)-1
        while i<j:
            if l[i] == l[j]:
                i+=1
                j-=1
            else:
                return False
        return True

File: 125-valid-palindrome/valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        
        StringBuilder word = new StringBuilder();
        for(int i=0;i<s.length();i++){
            if(Character.isLetterOrDigit(s.charAt(i))){
                char lowerCased = Character.toLowerCase(s.charAt(i));
                word.append(lowerCased);
            }
        }
        String modWord = word.toString();
        
        int i =0;
        int j=word.length()-1;

        // if(word.length()==1){
        //     return false;
        // }
        while(i<=j){
            if(word.charAt(i) != word.charAt(j)){
                return false;
            }
            i++;
            j--;
        }
        return true;


    }
}